card/src/main/java/com/adyen/checkout/card/CardComponent.kt

```diff
@@ -201,6 +204,10 @@ class CardComponent private constructor(
         return if (cardNumber.length < BIN_VALUE_LENGTH) cardNumber else cardNumber.substring(0..BIN_VALUE_LENGTH)
     }
 
+    private fun getLastFourDigitsFromCardNumber(cardNumber: String): String {
```

Comment:
do you think this needs a separate method?

Reply:
Hmm maybe not, i had more logic inside it first then i noticed it's not needed, will fix 👍